### DISCUSSION TITLE
Viewer for Cyrillic

### DISCUSSION THREAD

**Discussion by StuartZaq** [KDE/ghostwriter#660] [General]
The viewer does not display all the text if it is typed in Cyrillic

**Comment by StuartZaq** [comment]
![изображение](https://user-images.githubusercontent.com/32178831/133623080-2f0baa65-85a4-4009-9225-f525577ba004.png)

**Reply by unixba** [reply]
It seems that the problem is in the way non-ASCII (UTF-8) text is handled when rendering the HTML preview. The picture below illustrates the problem.

![Ghostwriter-2 0 1-non-ASCII-HTML-preview-BUG](https://user-images.githubusercontent.com/10323139/138353175-674cc515-7543-421e-a86e-be5ffe4ede28.png)

**Comment by XQryakusX** [comment]
It making lot of troubles, when you need write a massive text without preview, or preview isn't full. 
![angry-mad](https://user-images.githubusercontent.com/98166272/150537869-94c80736-ff4d-4538-aeba-67b8d6251565.gif)

**Comment by StuartZaq** [comment]
Installed release 2.1.6 and cyrillics now showed perfectly in preview window.
